examples/quick-start/bentoml-quick-start-guide.ipynb

```diff
@@ -152,9 +236,104 @@
+      "Requirement already satisfied, skipping upgrade: MarkupSafe>=0.23 in /Users/leon.sun/anaconda3/envs/IrisClassifier_2019_08_08_e1f4376a/lib/python3.7/site-packages (from Jinja2>=2.10.1->flask->bentoml[api_server]==0.3.1->-r /tmp/bentoml_archive/IrisClassifier/2019_08_08_e1f4376a/condaenv.ucnr9t3b.requirements.txt (line 1)) (1.1.1)\n",
+      "\n",
+      "#\n",
+      "# To activate this environment, use\n",
+      "#\n",
+      "#     $ conda activate IrisClassifier_2019_08_08_e1f4376a\n",
+      "#\n",
+      "# To deactivate an active environment, use\n",
+      "#\n",
+      "#     $ conda deactivate\n",
+      "\n"
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "no change     /Users/leon.sun/anaconda3/condabin/conda\n",
+      "no change     /Users/leon.sun/anaconda3/bin/conda\n",
+      "no change     /Users/leon.sun/anaconda3/bin/conda-env\n",
+      "no change     /Users/leon.sun/anaconda3/bin/activate\n",
+      "no change     /Users/leon.sun/anaconda3/bin/deactivate\n",
+      "no change     /Users/leon.sun/anaconda3/etc/profile.d/conda.sh\n",
+      "no change     /Users/leon.sun/anaconda3/etc/fish/conf.d/conda.fish\n",
+      "no change     /Users/leon.sun/anaconda3/shell/condabin/Conda.psm1\n",
+      "no change     /Users/leon.sun/anaconda3/shell/condabin/conda-hook.ps1\n",
+      "no change     /Users/leon.sun/anaconda3/lib/python3.7/site-packages/xonsh/conda.xsh\n",
+      "no change     /Users/leon.sun/anaconda3/etc/profile.d/conda.csh\n",
+      "no change     /Users/leon.sun/.bash_profile\n",
+      "No action taken.\n",
+      "/Users/leon.sun/anaconda3/envs/IrisClassifier_2019_08_08_e1f4376a/lib/python3.7/site-packages/sklearn/base.py:306: UserWarning: Trying to unpickle estimator SVC from version 0.20.3 when using version 0.21.3. This might lead to breaking code or invalid results. Use at your own risk.\n",
+      "  UserWarning)\n",
+      " * Serving Flask app \"IrisClassifier\" (lazy loading)\n",
+      " * Environment: production\n",
+      "\u001b[31m   WARNING: This is a development server. Do not use it in a production deployment.\u001b[0m\n",
+      "\u001b[2m   Use a production WSGI server instead.\u001b[0m\n",
+      " * Debug mode: on\n",
+      " * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)\n",
+      " * Restarting with stat\n",
+      "/Users/leon.sun/anaconda3/envs/IrisClassifier_2019_08_08_e1f4376a/lib/python3.7/site-packages/sklearn/base.py:306: UserWarning: Trying to unpickle estimator SVC from version 0.20.3 when using version 0.21.3. This might lead to breaking code or invalid results. Use at your own risk.\n",
+      "  UserWarning)\n",
+      " * Debugger is active!\n",
+      " * Debugger PIN: 130-753-966\n"
+     ]
+    }
+   ],
    "source": [
-    "!bentoml serve {saved_path}"
+    "!bentoml serve {saved_path} --with-conda"
```

Comment:
let's remove it here just to make sure quickstart guide runs in google colab - I will add more documentation and examples for --with-conda later on

Reply:
Yes, mis-submitted this file...